src/stacks-hierarchy/Root_ChooseTicketRecipientScreen/components/SubmitButton.tsx

```diff
@@ -88,10 +88,16 @@ export const SubmitButton = ({
       return;
     }
 
-    if (settingName && !name) {
-      setIsSubmitting(false);
-      onError('missing_recipient_name');
-      return;
+    if (settingName) {
```

Comment:
Could move `setIsSubmitting(false);` and `return;` outside the inner ifs

Reply:
Not really sure about that since we don't want to stop submitting if neither of the two ifs are true.

Reply:
Hmm right